frame/ddc-metrics-offchain-worker/src/lib.rs

```diff
@@ -21,25 +21,25 @@ use frame_system::offchain::{
 use hex_literal::hex;
 use pallet_contracts;
 use sp_core::crypto::KeyTypeId;
-use sp_core::offchain::StorageKind::PERSISTENT;
-use sp_io::offchain::local_storage_get;
 use sp_runtime::{
     offchain::{http, storage::StorageValueRef, Duration},
     traits::StaticLookup,
     AccountId32,
 };
-use sp_std::{str::from_utf8, vec::Vec};
+use sp_std::vec::Vec;
 
 #[macro_use]
 extern crate alloc;
 
 use alloc::string::String;
 
-pub const BLOCK_INTERVAL: u32 = 200; // TODO: Change to 1200 later [1h]. Now - 200 [10 minutes] for testing purposes.
+pub const BLOCK_INTERVAL: u32 = 100; // TODO: Change to 1200 later [1h]. Now - 200 [10 minutes] for testing purposes.
```

Comment:
Is that on purpose?

Reply:
Yes, this had 200 for block time = 3s. Since we increased it to 6s we need to change this value.